app/src/main/res/menu/bookmarks_edit.xml

```diff
@@ -4,6 +4,15 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/save_bookmark_button"
+        android:icon= "@drawable/ic_check_black_24dp"
+        app:iconTint="?primaryText"
+        android:title="Save"
```

Comment:
We need to move the title into a string resource in strings.xml so it can be localized